Chart.yaml

```
@@ -15,12 +15,12 @@ type: application
 # This is the chart version. This version number should be incremented each time
 # you make changes to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 5.4.0
+version: 5.4.1
```

Since we've changed a few things, in between, this should be:

```suggestion
version: 5.5.1
```